lib/github_service/commands/cross_repo_test.rb

```diff
@@ -274,15 +274,15 @@ def update_github_workflow_yaml_content
         raw_yaml = rugged_repo.blob_at(branch_ref.target.oid, ".github/workflows/ci.yaml").content
         content  = YAML.safe_load(raw_yaml)
 
-        content["jobs"] = test_repos.each_with_object({}) do |test_repo, result|
-          result[test_repo] = {
+        content["jobs"] = {
+          "cross-repo" => {
             "uses" => "ManageIQ/manageiq-cross_repo/.github/workflows/manageiq_cross_repo.yaml@master",
             "with" => {
-              "test-repo" => test_repo,
+              "test-repo" => test_repos.inspect,
```

Comment:
I wonder if we should change the key to test-repos to make it clear what is incoming

Reply:
Yeah we can do that I'll just need to push another PR to manageiq-cross_repo

Reply:
https://github.com/ManageIQ/manageiq-cross_repo/pull/86 https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/583